lib/taskana-core-test/src/test/java/acceptance/taskcomment/delete/DeleteTaskCommentAccTest.java

```diff
@@ -0,0 +1,161 @@
+import pro.taskana.workbasket.api.models.Workbasket;
+
+@TaskanaIntegrationTest
+public class DeleteTaskCommentAccTest {
+
+  private static final String ACCESS_USER_1_1_ID = "user-1-1";
+
+  @TaskanaInject TaskService taskService;
+  @TaskanaInject ClassificationService classificationService;
+  @TaskanaInject WorkbasketService workbasketService;
+
+  Classification defaultClassification;
+  Workbasket defaultWorkbasket;
+  Task task1;
+  TaskComment comment1;
+  TaskComment comment2;
+
+  @WithAccessId(user = ACCESS_USER_1_1_ID)
+  @BeforeEach
+  public void createTaskAndTaskComments() throws Exception {
+    task1 =
+        TaskBuilder.newTask()
+            .classificationSummary(defaultClassification.asSummary())
+            .workbasketSummary(defaultWorkbasket.asSummary())
+            .primaryObjRef(DefaultTestEntities.defaultTestObjectReference().build())
+            .buildAndStore(taskService);
+    comment1 =
+        TaskCommentBuilder.newTaskComment()
+            .taskId(task1.getId())
+            .textField("Text1")
+            .created(Instant.now())
+            .modified(Instant.now())
+            .buildAndStore(taskService);
+    comment2 =
+        TaskCommentBuilder.newTaskComment()
+            .taskId(task1.getId())
+            .textField("Text1")
+            .created(Instant.now())
+            .modified(Instant.now())
+            .buildAndStore(taskService);
+    List<TaskComment> taskComments = taskService.getTaskComments(task1.getId());
+    assertThat(taskComments).hasSize(2);
+  }
+
+  @WithAccessId(user = "admin")
+  @AfterEach
+  public void deleteTask() throws Exception {
+    taskService.cancelTask(task1.getId());
+    taskService.deleteTask(task1.getId());
+  }
+
+  @WithAccessId(user = "admin")
+  @BeforeAll
+  void setup() throws Exception {
+    defaultClassification =
+        DefaultTestEntities.defaultTestClassification().buildAndStore(classificationService);
+    defaultWorkbasket =
+        DefaultTestEntities.defaultTestWorkbasket().buildAndStore(workbasketService);
+    WorkbasketAccessItemBuilder.newWorkbasketAccessItem()
+        .workbasketId(defaultWorkbasket.getId())
+        .accessId(ACCESS_USER_1_1_ID)
+        .permission(WorkbasketPermission.OPEN)
+        .permission(WorkbasketPermission.READ)
+        .permission(WorkbasketPermission.APPEND)
+        .buildAndStore(workbasketService);
+  }
+
+  @WithAccessId(user = ACCESS_USER_1_1_ID)
+  @Test
+  void should_DeleteTaskComment_For_TaskCommentId() throws Exception {
+
+    taskService.deleteTaskComment(comment1.getId());
+
+    List<TaskComment> taskCommentsAfterDeletion = taskService.getTaskComments(task1.getId());
+    assertThat(taskCommentsAfterDeletion).hasSize(1);
+  }
+
+  @WithAccessId(user = "user-1-2", groups = "user-1-1") // to read comments
+  @Test
+  void should_FailToDeleteTaskComment_When_UserHasNoAuthorization() throws Exception {
+    assertThatThrownBy(() -> taskService.deleteTaskComment(comment1.getId()))
```

Comment:
Thrown Exception should be NotAuthorizedOnTaskCommentException, and follow the testing guideline of Exception